.buildkite/pipeline-nightly-e2e-tests.yml

```
@@ -12,7 +12,7 @@ steps:
           - .ci/setenvconfig e2e/aks
           - make -C .ci TARGET="run-deployer e2e-run" ci
         artifact_paths:
-          - "eck-diagnostic-*.zip"
+          - "eck-diagnostics-*.zip"
```

```suggestion
          - "eck-diagnostic*.zip"
```


I think we create `eck-diagnostic-*.zip` in test/e2e/test/diagnostics.go:
```go
run(ctx, "eck-diagnostics", "--output-directory", "/tmp", "-n", fmt.Sprintf("eck-diagnostic-%s.zip", normalizedTestName), "-o", testCtx.Operator.Namespace, "-r", strings.Join(otherNS, ","), "--run-agent-diagnostics")
```